### PR TITLE
added text-utilities Component

### PR DESCRIPTION
**Pull request by omkarpatke**
Done with all text utility components

### CODE REVIEW

File: componentsPage.html [hunk 1]
```diff
@@ -214,6 +215,55 @@ <h2>Icon Buttons</h2>
...
+               <h2>Heading</h2>
+               <p>Our Component library has all headings from <span>h1</span> to <span>h6</span></p>
+               <div class="headings-container">
+                   <h1 class="heading">H1 - Heading</h1>
+                   <h2>H2 - Heading</h2>
+                   <h3>H3 - Heading</h3>
+                   <h4>H4 - Heading</h4>
+                   <h5>H5 - Heading</h5>
+                   <h6>H6 - Heading</h6>
+               </div>
```
Comment (Naman-Saxena1) [inline componentsPage.html:229]: You have used class="heading" only on 1 element.  Not sure if it is intentional or you forgot to add on others.

File: componentsPage.html [hunk 2]
```diff
@@ -214,6 +215,55 @@ <h2>Icon Buttons</h2>
...
+               <h1>Text Utilities</h1>
+               <h2>Heading</h2>
+               <p>Our Component library has all headings from <span>h1</span> to <span>h6</span></p>
+               <div class="headings-container">
+                   <h1 class="heading">H1 - Heading</h1>
+                   <h2>H2 - Heading</h2>
+                   <h3>H3 - Heading</h3>
+                   <h4>H4 - Heading</h4>
+                   <h5>H5 - Heading</h5>
+                   <h6>H6 - Heading</h6>
```
Comment (rahulk31) [inline componentsPage.html:228]: Hey Omkar, I have a suggestion like to have separate class names for different headings.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: componentsPage.html (hunk 2 above)
Comment (rahulk31) [inline componentsPage.html:228]: Hey Omkar, I have a suggestion like to have separate classnames for each heading type with your own font-sizes and not depending on the default sizes of h1, h2, h3 and so on. 
You can have each headings in your primary and secondary colors.
eg: 
h1-primary(h1 blue)
h1-secondary(h1 black)

File: componentsPage.html [hunk 3]
```diff
@@ -214,6 +215,55 @@ <h2>Icon Buttons</h2>
...
+               src="https://carbon.now.sh/embed?bg=rgba%28175%2C191%2C211%2C0.97%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253EH1%2520-%2520Heading%253C%252Fh1%253E%250A%253Ch2%253EH2%2520-%2520Heading%253C%252Fh2%253E%250A%253Ch3%253EH3%2520-%2520Heading%253C%252Fh3%253E%250A%253Ch4%253EH4%2520-%2520Heading%253C%252Fh4%253E%250A%253Ch5%253EH5%2520-%2520Heading%253C%252Fh5%253E%250A%253Ch6%253EH6%2520-%2520Heading%253C%252Fh6%253E"
+               style="width: 327px; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+               sandbox="allow-scripts allow-same-origin">
+               </iframe>
+
+             <h2>Small Text ( Paragraph, Span )</h2>
+             <p>Both Paragraph and Span has same font style. for paragraph you can use <span>p</span> tag and for <span>span</span> you can use span tag.</p>
+             <div class="content-container">
+                 <p>P - Paragraph</p>
+                 <span> Span Tag </span>
```
Comment (rahulk31) [inline componentsPage.html:241]: Similarly, for paragraph and span too.

**Review by rahulk31 — COMMENTED** [review]
Rest looks great to me. Readable code with neat design. Great work 🚀

File: componentsPage.html (hunk 1 above)
Comment (omkarpatke) [reply componentsPage.html:229]: actually, this is left by mistake.

File: componentsPage.html (hunk 2 above)
Comment (omkarpatke) [reply componentsPage.html:228]: btw great idea I will work on it.

File: componentsPage.html (hunk 3 above)
Comment (omkarpatke) [reply componentsPage.html:241]: ok...

**Pull request merged**